openedx/core/djangoapps/content/search/api.py

```diff
@@ -17,35 +17,37 @@
 from meilisearch import Client as MeilisearchClient
 from meilisearch.errors import MeilisearchApiError, MeilisearchError
 from meilisearch.models.task import TaskInfo
-from opaque_keys.edx.keys import UsageKey, OpaqueKey
-from opaque_keys.edx.locator import LibraryContainerLocator, LibraryLocatorV2, LibraryCollectionLocator
+from opaque_keys import OpaqueKey
+from opaque_keys.edx.keys import UsageKey
+from opaque_keys.edx.locator import LibraryCollectionLocator, LibraryContainerLocator, LibraryLocatorV2
 from openedx_learning.api import authoring as authoring_api
-from common.djangoapps.student.roles import GlobalStaff
 from rest_framework.request import Request
+
 from common.djangoapps.student.role_helpers import get_course_roles
+from common.djangoapps.student.roles import GlobalStaff
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
-from openedx.core.djangoapps.content.search.models import get_access_ids_for_request, IncrementalIndexCompleted
 from openedx.core.djangoapps.content.search.index_config import (
     INDEX_DISTINCT_ATTRIBUTE,
     INDEX_FILTERABLE_ATTRIBUTES,
-    INDEX_SEARCHABLE_ATTRIBUTES,
-    INDEX_SORTABLE_ATTRIBUTES,
     INDEX_RANKING_RULES,
+    INDEX_SEARCHABLE_ATTRIBUTES,
+    INDEX_SORTABLE_ATTRIBUTES
 )
+from openedx.core.djangoapps.content.search.models import IncrementalIndexCompleted, get_access_ids_for_request
 from openedx.core.djangoapps.content_libraries import api as lib_api
 from xmodule.modulestore.django import modulestore
 
 from .documents import (
     Fields,
     meili_id_from_opaque_key,
-    searchable_doc_for_course_block,
+    searchable_doc_collections,
     searchable_doc_for_collection,
     searchable_doc_for_container,
+    searchable_doc_for_course_block,
     searchable_doc_for_library_block,
     searchable_doc_for_key,
-    searchable_doc_collections,
     searchable_doc_tags,
-    searchable_doc_tags_for_collection,
+    searchable_doc_tags_for_collection
```

Comment:
nit: should keep the trailing `,` here, I'm surprised pylint didn't complain :)
```suggestion
    searchable_doc_tags_for_collection,
```